src/style.scss

```diff
@@ -123,3 +123,136 @@ header {
     margin-left: 20px;
   }
 }
+
+.pop {
+  display: none;
+  position: fixed;
+  top: 20;
+  bottom: 0;
+  left: 20px;
+  max-height: 100vh;
+  min-width: 100vw;
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
```

Comment:
- [ ] Please consider disabling `scroll ` on the main page when the pop-up is active. This will allow the user to only focus on the content of the pop-up.
- [ ] You may also consider reducing the pop-up's width to show its outline clearly.
- [ ] I've noticed you're rendering the modal data on top of the blur. That is interfering with the clarity of the content. I think that may be changed by giving the modal a background of its own and leaving the blur effect around it